azure-py-functions/README.md

```diff
@@ -0,0 +1,182 @@
+# Deploying Serverless Applications with Azure Functions
+
+You will deploy a Azure Function Apps with HTTP-triggered serverless functions in python
+
+## Running the App
+
+1.  Login to Azure CLI (you will be prompted to do this during deployment if you forget this step):
+
+    ```
+    $ az login
+    ```
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init dev
+    ```
+
+
+1. Create a virtual environment and install python dependencies
+
+    ### Windows
+    Run the following command to create a virtual environment
+    ```bash
```

Comment:
is this required now? I thought the CLI handled this for us?

Reply:
We had this here due to a python workshop where the team was windows and ran into a lot of issues.